API/fleece/Expert.hh

```diff
@@ -0,0 +1,193 @@
+//
+// Expert.hh
+//
+// Copyright 2017-Present Couchbase, Inc.
+//
+// Use of this software is governed by the Business Source License included
+// in the file licenses/BSL-Couchbase.txt.  As of the Change Date specified
+// in that file, in accordance with the Business Source License, use of this
+// software will be governed by the Apache License, Version 2.0, included in
+// the file licenses/APL2.txt.
+//
+
+#pragma once
+#ifndef _FLEECE_OBSCURE_HH
+#define _FLEECE_OBSCURE_HH
+#ifndef _FLEECE_HH
+#include "Fleece.hh"
+#endif
+
+#include "FLExpert.h"
+
+FL_ASSUME_NONNULL_BEGIN
+
+namespace fleece {
+
+    //  Rarely-needed or advanced functionality; C++ wrapper around fleece/Obscure.h
```

Comment:
Wrapped around what?  Should this be FLExpert.h ?

Reply:
oops, yeah, it was briefly called Obscure.h before I went with "expert".